### PR TITLE
:sparkles: Add download buttons for report

### LINKED ISSUE #613: Download CSV
https://issues.redhat.com/browse/MTA-116

### PR DESCRIPTION
**Pull request by ibolton336**
Closes #613 #651

### CODE REVIEW

File: client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-analysis.tsx [hunk 1]
```diff
@@ -118,6 +118,21 @@ export const ApplicationListExpandedAreaAnalysis: React.FC<
             "Not available"
           )}
         </DescriptionListDescription>
+        {task?.state === "Succeeded" && (
+          <DescriptionListDescription>
+            <Tooltip content="Click to download Analysis report">
+              <Button variant="link" isInline>
+                <Link
+                  to={`/hub/applications/${application.id}/bucket${task?.data?.output}`}
```
Comment (gildub) [inline client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-analysis.tsx:126]: We probably want to use the defined path APPLICATIONS here.

**Review by gildub — COMMENTED** [review]
Don't we need to test for global settings allowing CSV download ?

File: client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-analysis.tsx (hunk 1 above)
Comment (ibolton336) [reply client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-analysis.tsx:126]: Good idea :+1:

**Review by gildub — COMMENTED** [review]
It's hard to visually distinct the html link from the csv one. 
Not sure what are the requirements meanwhile it might worth adding a comma or equivalent to clearly separate them ?

CSV link return html file.

I18n is missing.

**Review by gildub — APPROVED** [review]
I'm still facing a html file instead of tar file for csv but that's no blocker so we can address it later.

**Pull request merged**